database/generateBookedDates.js

```diff
@@ -0,0 +1,60 @@
+const randomNumberUpTo = (limit) => Math.floor(Math.random() * limit);
+
+module.exports = function generateBookedDates(minStay, listing_id) {
+
+  // create objects to map months to corresponding number of days in month and to year
+  let months = [11,12,1,2];
+  let daysInMonth = {
+    11: 30,
+    12: 31,
+    1: 31,
+    2: 28
+  }
+  let monthToYear = {
+    11: 2018,
+    12: 2018,
+    1: 2019,
+    2: 2019
+  }
+
+  // create array of all possible days for booking (all days Nov '18 through Feb '19)
+  let days = [];
+  for (let month of months) {
+    for (let day = 1; day < daysInMonth[month] + 1; day++) {
+      dayString = day.toString()
+      if (dayString.length === 1) {
+        dayString = '0' + dayString;
+      }
+      monthString = month.toString()
+      if (monthString.length === 1) {
+        monthString = '0' + monthString;
+      }
+      days.push(`${monthToYear[month]}-${monthString}-${dayString}`)
+    }
+  }
```

Comment:
I know this is not actually new code, but just reiterating that if you do refactor this to use Moment.js, it'll be easy to get string ISO dates and do a concise loop on some condition like `while (date.isBefore('2019-02-28'))`. http://momentjs.com/docs/#/query/is-before/

Not suggesting the refactoring for this pull request though, just in case you want to refactor later.

Reply:
Thanks for the reminder.